client/my-sites/comments/comment-list/index.jsx

```diff
@@ -442,15 +459,57 @@ const mapStateToProps = ( state, { siteId, status } ) => {
 };
 
 const mapDispatchToProps = ( dispatch, { siteId } ) => ( {
-	changeCommentStatus: ( commentId, postId, status ) => dispatch( changeCommentStatus( siteId, postId, commentId, status ) ),
+	changeCommentStatus: ( commentId, postId, status, analytics = { alsoUnlike: false, isUndo: false } ) => dispatch( withAnalytics(
```

Comment:
I believe analytics props should be `snake_case`. eg `also_unlike`. I'm not sure if we record correctly other